src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -23,11 +23,15 @@ export class DictionaryOverview extends Component {
     fetchDictionaryConcepts: propTypes.func.isRequired,
     fetchVersions: propTypes.func.isRequired,
     createVersion: propTypes.func.isRequired,
-    error: propTypes.array.isRequired,
+    error: propTypes.array,
```

Comment:
User `arrayOf()` here.

Reply:
@dessHub , the errors receive have varying fields so we can't specify the exact one here.